content/usage/secrets/registries.md

```diff
@@ -15,6 +15,21 @@ Drone provides the ability to store registry credentials. These credentials can
 These credentials are never exposed to your pipeline, which means they cannot be used to push, and are safe to use with pull requests, for example. Pushing to a registry still require setting credentials for the appropriate plugin.
 {{% /alert %}}
 
+# Image Caching Behavior
+
+{{% alert error %}}
+Pull private images with caution.
+{{% /alert %}}
+
+All images (including private images) are pulled and cached by the Docker daemon.
+Drone assumes that any image in the local cache can be used as a build image.
```

Comment:
I think maybe just clarify this a bit more that this is the default docker behavior, and that drone does not change this default behavior to enforce any additional access controls

```diff
- Drone assumes that any image in the local cache can be used as a build image.
+ Neither Docker nor Drone restrict the use of cached images.
```

Reply:
Sounds good!